go-controller/pkg/ovn/address_set/fake_address_set.go

```diff
@@ -272,6 +273,28 @@ func (as *fakeAddressSets) AddIPs(ips []net.IP) error {
 	return nil
 }
 
+// TEMP
+func (as *fakeAddressSets) AddIPsReturnsOps(ips []net.IP) ([]ovsdb.Operation, error) {
+	var ops []ovsdb.Operation
+	var err error
+	as.Lock()
+	defer as.Unlock()
+
+	for _, ip := range ips {
+		if utilnet.IsIPv6(ip) {
+			ops, err = as.ipv6.addIPsReturnsOps(ip)
+		} else {
+			ops, err = as.ipv4.addIPsReturnsOps(ip)
```

Comment:
@tssurya this will only work for single-stack, I think you want to `ops = append(ops, <something>)`?